src/index.js

```diff
@@ -17,6 +17,7 @@ export const defaultOptions = {
 	deterministic: false,
 	enableOptimizations: false,
 	enableTooltip: true,
+	tooltipProps: {},
```

Comment:
let's rename this to `tooltipOptions` (I know it's a little weird that `tooltipOptions` is in `options`), but I think it's a little better than `*Props`, where the `props` naming usually refers to component-based properties.  In this case, we're not really dealing with component-based properties or interfaces.

Reply:
I was in two minds about this too -- I followed using Tippy's terminology (which admittedly is an odd use of Props) but I'm not fussed either way.  `tooltipOptions` sounds suitably agnostic :+1: